changelog.txt

```diff
@@ -36,6 +36,14 @@ Template for new versions:
 
 ## Removed
 
+# 50.13-r2
+
+## New Features
+- `gui/autobutcher`: add functionality to butcher and unbutcher all animals
```

Comment:
this doesn't quite qualify as a "new feature"; it should be in the Misc Improvements section instead